python_modules/dagster/dagster/core/storage/migration/utils.py

```diff
@@ -200,6 +200,10 @@ def create_event_log_event_idx():
     if not has_table("event_logs"):
         return
 
+    indices = [x.get("name") for x in get_inspector().get_indexes("event_logs")]
```

Comment:
Adds a missing idempotency check for creating an index in one of the repeated migrations.  Caught by our backcompat tests.